dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -63,8 +65,23 @@ public ClassReferenceGraph getClassReferenceGraph() {
     return classReferenceGraph;
   }
 
+  @VisibleForTesting
+  static LinkageChecker create(List<ClassPathEntry> classPath) throws IOException {
+    return create(classPath, ImmutableSet.copyOf(classPath), null);
+  }
+
+  /**
+   * Returns Linkage Checker for {@code classPath}.
+   *
+   * @param classPath List of JAR files to find linkage errrors
+   * @param entryPoints List of JAR files to define entry point classes in reachability.
+   * @param exclusionFile Exclusion file to filter out linkage errors.
```

Comment:
exclusion file to suppress linkage errors.

Reply:
Updated.